helm/vertical-pod-autoscaler-app/crds/vpa-v1-crd.yaml

```diff
@@ -1,94 +1,800 @@
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
```

Comment:
Hello CRD update :wave: 

Ok this is not nice, I had to deal with the same mess for prometheus-operator recently. Let's chat.

Basically we need:

1. Move the CRD into its own chart/repo
2. Setup a job in this repo to fake CRD ownership for helm

We need this because, helm does not upgrade CRD.

Reply:
Solved, we now have a crd-patch job.